### ISSUE TITLE
Test failure - sanitising SVG

### ISSUE THREAD

**Issue by takluyver** [jupyter/nbconvert#545]
This cropped up on Jenkins:

```
+ py.test --pyargs nbconvert
============================= test session starts ==============================
platform linux -- Python 3.5.2, pytest-3.0.6, py-1.4.32, pluggy-0.4.0
rootdir: /tmp/tmp.VWneV6z9SB, inifile:
collected 222 items

. .....................................................................................................................................................................F........................................................

=================================== FAILURES ===================================
_______________________ TestSanitizer.test_svg_handling ________________________

self = <nbconvert.preprocessors.tests.test_sanitize.TestSanitizer testMethod=test_svg_handling>

    def test_svg_handling(self):
        """
            Test to make sure that svgs are handled 'properly'

            We only allow <img> tags (via markdown syntax) and not all the other ways
            to embed svg: <object>, <embed>, <iframe> nor inline <svg>
            """
        preprocessor = self.build_preprocessor()
        preprocessor.strip = True

        self.assertEqual(
            self.preprocess_source(
                'markdown',
                """
                    ![some image](http://example.com/something.svg)
                    <object data="something.svg" type="image/svg+xml" />
                    <embed data="something.svg" type="image/svg+xml" />
                    <iframe src="http://example.com/something.svg" />
                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 68 65">
                        <path fill="#1A374D" d="M42 27v-20c0-3.7-3.3-7-7-7s-7 3.3-7 7v21l12 15-7 15.7c14.5 13.9 35 2.8 35-13.7 0-13.3-13.4-21.8-26-18zm6 25c-3.9 0-7-3.1-7-7s3.1-7 7-7 7 3.1 7 7-3.1 7-7 7z"/>
                        <path d="M14 27v-20c0-3.7-3.3-7-7-7s-7 3.3-7 7v41c0 8.2 9.2 17 20 17s20-9.2 20-20c0-13.3-13.4-21.8-26-18zm6 25c-3.9 0-7-3.1-7-7s3.1-7 7-7 7 3.1 7 7-3.1 7-7 7z"/>
                    </svg>
                    """,
                preprocessor
            ).strip(),
            """
                ![some image](http://example.com/something.svg)
>               """.strip(),
        )
E       AssertionError: '![so[38 chars].svg)\n                \n                \n   [522 chars]&gt;' != '![so[38 chars].svg)'
E       Diff is 724 characters long. Set self.maxDiff to None to see it.

/var/lib/jenkins/shiningpanda/jobs/92f5a430/virtualenvs/d41d8cd9/lib/python3.5/site-packages/nbconvert/preprocessors/tests/test_sanitize.py:59: AssertionError
==================== 1 failed, 221 passed in 144.63 seconds ====================
```

**Comment by michaelpacer** [comment]
This looks like the same error that is cropping up here: https://github.com/jupyter/nbconvert/pull/547.

**Comment by minrk** [comment]
I neglected to look for open issues, sorry. I believe this was closed by #548.
